conda-recipes/qsharp/meta.yaml

```diff
@@ -13,12 +13,12 @@ requirements:
     - python
     - setuptools
     - jupyter
-    - jupyter_client
+    - jupyter_client<5.3.3
     - iqsharp={{ version }}
 
   run:
     - python
-    - jupyter_client
+    - jupyter_client<5.3.3
```

Comment:
It would be lovely if these version numbers were (possibly environment?) variables all set in one place.